### PR TITLE
[CP-3004] Added helper components for contacts list selection

### PR DESCRIPTION
**Pull request by mkurczewski**
JIRA Reference: [CP-3004]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-3004]: https://appnroll.atlassian.net/browse/CP-3004?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/generic-view/ui/src/lib/interactive/format-message.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { selectActiveDeviceConfiguration } from "generic-view/store"
+
+export const FormatMessage: APIFC<FormatMessageData, FormatMessageConfig> = ({
+  data,
+  config,
+  children,
+  ...props
+}) => {
+  const id = useId()
+  const deviceConfig = useSelector(selectActiveDeviceConfiguration)
```
Comment (dkarski) [inline libs/generic-view/ui/src/lib/interactive/format-message.tsx:25]: ⚪ 
Is it necessary to tie this component to the active device state? Should lang come from the API or a default configuration Provider instead?

File: libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-title-icon.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const ModalTitleIcon: APIFC<undefined, ModalTitleIconConfig> = ({
   children,
   ...rest
 }) => {
+  // @ts-ignore
```
Comment (dkarski) [inline libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-title-icon.tsx:17]: Why was this @ts-ignore added? I removed it locally, and TypeScript doesn't throw any errors 🤷‍♂️

**Review by dkarski — APPROVED** [review]

File: libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-title-icon.tsx (hunk 2 above)
Comment (mkurczewski) [reply libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-title-icon.tsx:17]: It doesn't throw until... it throws. Sometimes during hot-reload you can get TS error from here in the console, while in IDE everything works fine 🤷‍♂️ . I tried to fix it but couldn't find the real issue, so I added the ignore command to not be forced to restart the app every time it happens.

File: libs/generic-view/ui/src/lib/interactive/format-message.tsx (hunk 1 above)
Comment (mkurczewski) [reply libs/generic-view/ui/src/lib/interactive/format-message.tsx:25]: For now only components are generated only in context of active device.

**Pull request merged**